cocos/renderer/gfx-gles2/GLES2Commands.cpp

```diff
@@ -1031,7 +1020,7 @@ void cmdFuncGLES2CreateShader(GLES2Device *device, GLES2GPUShader *gpuShader) {
         gpuShader->glSamplerTextures.resize(gpuShader->samplerTextures.size());
 
         for (size_t i = 0; i < gpuShader->glSamplerTextures.size(); ++i) {
-            UniformSamplerTexture &        samplerTexture    = gpuShader->samplerTextures[i];
+            UniformSamplerTexture         &samplerTexture    = gpuShader->samplerTextures[i];
```

Comment:
These changes are probably come from the wrong version of LLVM. Try installing 12.0 for now.